libnavigation-core/src/main/java/com/mapbox/navigation/core/trip/session/MapboxTripSession.kt

```diff
@@ -705,4 +708,13 @@ internal class MapboxTripSession(
             action(voiceInstructions)
         }
     }
+
+    private fun BasicSetRoutesInfo.toSetRoutesReason(): SetRoutesReason {
```

Comment:
Let's cut a ticket to refactor this as mentioned in the comments above.

Reply:
NAVAND-891